community-content/prediction_featurestore_integration/prediction_featurestore_integration.ipynb

```diff
@@ -0,0 +1,1470 @@
+{
```

Comment:
<p>This there a API method for replicating feature stores? Let's try to keep the creation of FS code as minimal as possible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>